expression/util.go

```diff
@@ -357,3 +357,80 @@ func extractFiltersFromDNF(ctx sessionctx.Context, dnfFunc *ScalarFunction) ([]E
 	}
 	return extractedExpr, ComposeDNFCondition(ctx, newDNFItems...)
 }
+
+// GetFuncArgLen gets the length of the row.
+func GetFuncArgLen(e Expression) int {
```

Comment:
Will `GetRowFuncArgLen` be more accurate,
since `GetFunArg` at line 380 is not only used to get the args of `RowFunc`